cmd/d2/main.go

```diff
@@ -113,12 +113,28 @@ func run(ctx context.Context, ms *xmain.State) (err error) {
 	}
 	ms.Log.Debug.Printf("using layout plugin %s (%s)", envD2Layout, pluginLocation)
 
+	var pw *playwright.Playwright
+	var browser playwright.Browser
+	if filepath.Ext(outputPath) == ".png" {
+		pw, browser, err = png.InitPlaywright()
+		if err != nil {
+			return err
+		}
+		defer func() error {
```

Comment:
ur not doing anything with this error. maybe just ms.log.error it if cleanup returns an error